<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Shots</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="shotsLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Shots Dir   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="shotsFilePathBox">
        <property name="toolTip">
         <string>Add directory where you see list of shots</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton2">
        <property name="toolTip">
         <string>Browse for directory</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CSV File    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="csvFilePathBox">
        <property name="toolTip">
         <string>Add path to CSV file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Output Dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputPathBox">
        <property name="toolTip">
         <string>Path to save maya files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton1">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="showDetailsButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Show Details</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hideDetailsButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Hide Details</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="statusBox">
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="filesOnlyButton">
        <property name="toolTip">
         <string>Create maya files only, don't create collage</string>
        </property>
        <property name="text">
         <string>Files Only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="collageOnlyButton">
        <property name="toolTip">
         <string>Don't create maya shot files</string>
        </property>
        <property name="text">
         <string>Collage Only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="saveToLocalButton">
        <property name="toolTip">
         <string>Save shots to C:\Users\USER.NAME\Documents\create_shots</string>
        </property>
        <property name="text">
         <string>Save to Local</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>statusBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>statusBox</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>showDetailsButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>hideDetailsButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>showDetailsButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>hideDetailsButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filesOnlyButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>collageOnlyButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collageOnlyButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>filesOnlyButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
